projects/backend/controllers/render.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from 'express'
+import fetch from 'node-fetch'
+
+const fetchRenderedArticle = async (url: string) => {
+    const response = await fetch(url, {
+        headers: {
+            Accept:
+                'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+        },
+    })
+    const responseBody = await response.text()
+    return {
+        success: response.statusText === 'OK',
+        status: response.status,
+        body: responseBody,
+    }
+}
+
+// TODO: this needs a test once we're happy with the correct format for the paths
+const replaceImageUrls = (html: string): string => {
+    return html.replace(/https:\/\/i.guim.co.uk\/img\//g, `../media/`)
```

Comment:
we may not need the `../media` prefix but we can update this later once we have html bundle ready